app/views/settings/index.html.erb

```diff
@@ -28,6 +28,7 @@
         <h4 class="text-center">To use with other apps</h4>
         <h6 class="text-center">Copy & Paste the following into your app:</h6>
         <pre class="text-center">http://localhost:3000/api/v1/links/create?api_key=<%= @settings.api_key %>&url=%@</pre>
+        <%= button_to "Update api key", "settings/regenkey", class: "btn-lg btn-warning btn-block" %>
```

Comment:
instead of "settings/regenkey", use the rails path just in case we change it in the routes at sometime!

`button_to "Update api key", settings_regenkey_path, .....`

Also, you need to give the user a popup to warn them that they are changing their key!